### PR TITLE
⚡️ [Portfolio] Lazy-load metadata

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

**Review by williamchong — APPROVED** [review]

**Comment by williamchong** [comment]
Would this affect sorting though

**Review by WeiJun0827 — APPROVED** [review]

**Comment by nwingt** [comment]
seems yes...

**Comment by nwingt** [comment]
Integrated infinite scroll, NFT metadata, etc will be fetched in the portfolio page instead of the individual item, with a concurrent limit of 10 requests

File: src/pages/_id/index.vue [hunk 1]
```diff
@@ -140,17 +140,18 @@
...
-            ref="nftGrid"
-          >
+        <CardV2
+          v-if="isLoading"
+        >{{ $t('nft_portfolio_page_label_loading') }}</CardV2>
+        <div
+          v-else-if="
+            currentTab === 'collected' ||
+              // Show grid if empty in created tab & not user portfolio
+              (currentTab === 'created' && (sortedCreatedClassIds.length || (!sortedCreatedClassIds.length && getAddress === wallet)))
```
Comment (williamchong) [inline src/pages/_id/index.vue:150]:
```suggestion
              (currentTab === 'created' && (sortedCreatedClassIds.length || getAddress === wallet))
```
Is this equivalent?

File: src/mixins/portfolio.js [hunk 2]
```diff
@@ -15,6 +16,10 @@ const tabOptions = {
   created: 'created',
 };
 
+const ITEMS_PER_PAGE = 10;
+
+const fetchNFTInfoLimit = throat(10);
```
Comment (williamchong) [inline src/mixins/portfolio.js:21]: is this 10 related to ITEMS_PER_PAGE?

Comment (williamchong) [inline src/mixins/portfolio.js:21]: `fetchNFTInfoThrottled` or `throttledFetchNFTInfo` avoid ambiguity word `limit`

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/mixins/portfolio.js (hunk 2 above)
Comment (nwingt) [reply src/mixins/portfolio.js:21]: Not directly, it is the number of concurrent NFT metadata requests

Comment (nwingt) [reply src/mixins/portfolio.js:21]: How about `throttleNFTInfoFetch`?

**Review by williamchong — APPROVED** [review]
